src/components/Notice/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigate } from 'react-router-dom';
+
+import styled from 'styled-components';
+
+type HeaderProps = {
+    children: React.ReactNode;
```

Comment:
기존 컨벤션과 동일하게 맞추려면 이렇게 처리해줘야 할 것 같네요~
```suggestion
    children: ReactNode;
```